src/Form/AuthenticationForm.php

```diff
@@ -199,6 +220,8 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     ];
 
     $form['test_connection'] = [
+      '#prefix' => '<div id="apigee-edge-connection-info">',
```

Comment:
I usually prefer using`container` render element for creating the AJAX wrapper instead of prefix+suffix.
https://www.metaltoad.com/blog/avoiding-drupal-7-ajax-pitfalls
https://github.com/drupal/core/blob/17557c97a816bc78e70989a86064c562bc27027f/modules/editor/editor.module#L137